### ISSUE TITLE
Uploading larger layers fail

### ISSUE THREAD

**Issue by timlinux** [kartoza/geonode#52]
I tried to upload these files:

```
-rw-r--r--  1 timlinux  staff    63M Feb 23 11:05 buildings.dbf
-rw-r--r--  1 timlinux  staff   282B Feb 23 11:05 buildings.keywords
-rw-r--r--  1 timlinux  staff   596B Feb 23 11:05 buildings.license
-rw-r--r--  1 timlinux  staff   257B Feb 23 11:05 buildings.prj
-rw-r--r--  1 timlinux  staff    22K Feb 26 16:12 buildings.qml
-rw-r--r--  1 timlinux  staff    12M Feb 23 11:05 buildings.shp
-rw-r--r--  1 timlinux  staff   699K Feb 23 11:05 buildings.shx
-rw-r--r--  1 timlinux  staff    13K Feb 26 16:28 buildings.xml
```

Uploading failed with:

```
413 Request Entity Too Large
```

**Comment by lucernae** [comment]
I increased the size of 

client_max_body_size 500M

in geonode.kartoza.nginx.conf

**Comment by felix-yew** [comment]
Hi @timlinux and mas @lucernae 

I try to upload the training data to the geonode.inasafe.org, but some data failed to upload. Failed because entity too large.
![error upload buildings_entity too large](https://cloud.githubusercontent.com/assets/13621886/24340723/91d88004-12de-11e7-8703-a134f50bc167.png)
This totally size of building data 71.1 MB

And, then I find failed upload when I try to upload the raster population.
![error object upload large data like raster pop](https://cloud.githubusercontent.com/assets/13621886/24340788/181d33ee-12df-11e7-88e0-9aed1f556eba.png)
I check to the layer but there is no raster population layer has been uploaded. This totally size of raster population data are 149 MB

> increased the size of
client_max_body_size 500M
in geonode.kartoza.nginx.conf

this max size also support to the geonode.inasafe.org or not? 
* If yes, why the data that under 500M size can not upload properly? 
* If no, please make it happy :smile:

**Comment by ismailsunni** [comment]
Hi @felix-yew 
The one that @lucernae changed is the one for geonode.kartoza. It's different instance of geonode, and has different setting. I will see if I can set it to 500 MB also for geonode.inasafe.org.

**Comment by cchristelis** [comment]
Let's chunk the files during upload to avoid this problem.

**Comment by gubuntu** [comment]
@cchristelis @euclidian how is progress with chunking (using AT approach?)

Will this be applied to raster and vector uploads?

**Comment by cchristelis** [comment]
@gubuntu, yeah, the same approach. @euclidian seems to be having some difficulty finding his feet with the JS.

**Comment by gubuntu** [comment]
@NyakudyaA didn't you test demo.geonode.org and it doesn't fail with large file uploads? Did you come to any conclusions?

**Comment by NyakudyaA** [comment]
@gubuntu I tested the demo site and they have issues with uploading big datasets as well. I saw on the geonode list the geonode guys have it on their to-do list as well.

**Comment by NyakudyaA** [comment]
@gubuntu I tested the demo site and they have issues with uploading big datasets as well. I saw on the geonode list the geonode guys have it on their to-do list as well.
